### PR TITLE
PRO-82 Secondary Navigation

### PR DESCRIPTION
**Pull request by hemadri-tank-bacancy**
Points:
  1.  swipe left to see a list of all my assets
  2. swipe right, see upgraded QR code scanner
  3. Updated wallet connect

https://drive.google.com/file/d/1P9X7iS9AaPP6nt60XOcM-Us2-IR1ouFq/view

### CODE REVIEW

File: src/screens/WalletConnect/ConnectedApps/AppListItem.js [hunk 1]
```diff
@@ -19,60 +19,175 @@
...
+          </Text>
+        </RightAddOn>
+      </TouchableContainer>
+      <ConnectedAppsMenu
+        visible={visibleModal}
+        onHide={setVisibleModal}
+        onSelect={(item: itemProps) => {
+          if (item.value === 'Switch wallet') setVisibleWalletSwitchModal(true);
+          if (item.value === 'Switch network') setVisibleNetworkSwitchModal(true);
+          if (item.value === 'Disconnect') disconnect();
```
Comment (IAmKio) [inline src/screens/WalletConnect/ConnectedApps/AppListItem.js:154]: Would this work for different languages?

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few minor points but other than that - fantastic work! Really well done 🙏 🚀

File: src/components/Modals/ConnectedAppsModal/DropDown.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  onHide: (val: boolean) => void;
+  modalContent: any;
+}
+
+const { height } = Dimensions.get('window');
+
+const DropDown: FC<Props> = ({ dropDownStyle, visible, onHide, modalContent }) => {
+  const colors = useThemeColors();
+
+  const [ref, setRef] = useState(null);
```
[the remark this answers is not in the document]
Comment (hemadri-tank-bacancy) [reply src/components/Modals/ConnectedAppsModal/DropDown.tsx:51]: Actually here using ref is getting from another screen. so useRef is not helpful here.

File: src/screens/WalletConnect/ConnectedApps/AppListItem.js (hunk 1 above)
Comment (hemadri-tank-bacancy) [reply src/screens/WalletConnect/ConnectedApps/AppListItem.js:154]: Yes, this is working in different languages. label showing in app.

File: src/actions/walletConnectActions.js [hunk 3]
```diff
@@ -194,9 +201,28 @@ export const approveWalletConnectConnectorRequestAction = (peerId: string, chain
   };
 };
 
+export const updateWalletConnectConnectorSessionAction = (connector: Object, sessionData: sessionDataProps) => {
+  return (dispatch: Dispatch) => {
+    try {
+      connector.updateSession(sessionData);
+
+      dispatch({ type: UPDATE_WALLETCONNECT_SESSION, payload: { session: connector.session } });
+      // dispatch(subscribeToWalletConnectConnectorEventsAction(connector));
```
[the remark this answers is not in the document]
Comment (hemadri-tank-bacancy) [reply src/actions/walletConnectActions.js:210]: Removed!

File: index.js [hunk 4]
```diff
@@ -20,8 +20,10 @@
 
 import 'react-native-gesture-handler';
 import '@ethersproject/shims';
-import { AppRegistry } from 'react-native';
+import { AppRegistry, LogBox } from 'react-native';
 import App from './App';
 import { name as appName } from './app.json';
 
+LogBox.ignoreAllLogs();
```
[the remark this answers is not in the document]
Comment (hemadri-tank-bacancy) [reply index.js:27]: Removed!

File: src/components/Modals/ConnectedAppsModal/DropDown.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (IAmKio) [reply src/components/Modals/ConnectedAppsModal/DropDown.tsx:51]: Ah got it, thanks for the explanation

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**